android/gradle.properties

```diff
@@ -1,2 +1,17 @@
+## For more details on how to configure your build environment visit
```

Comment:
The changes in this file were added by Android Studio automatically to fix some warnings.

Reply:
Those changes look good and are sensible.